tools/debug/README.md

```diff
@@ -0,0 +1,36 @@
+# DebugSymbolViewer
+
+The Debug Symbol viewer will help the developer when a crash occurs.
+It will show some call stack information when the application has faced an abort. To be precise, it will show the symbols corresponding to the stack addresses present in the kernel text address range.
+
+### Prerequisites
+Install python3.8 & python 2.7, pyserial(at pip)
+
+### How to USE
+
+1. Copy crash logs  
+    First copy the crash logs to a file named log_file in tools/debug/`<log_file>`
+2. Run Script 
+    $ python3 ramdumpParser.py -e <Tinyara elf path>
+
+    ex)
+    $ python3 ramdumpParser.py -e ../../build/output/bin/tinyara.axf
+
+```
+*************************************************************
+dump_file         : None
+log_file          : None
+Tinyara_elf_file  : ../../build/output/bin/tinyara.axf
+*************************************************************
+
+~~~~~~~~~~~~~~~~~~~~~~~~  DEBUG DISPLAY SYMBOLS ~~~~~~~~~~~~~~~~~~~~~
+Address:  0xe005df8	Symbol:  test_func	/home/test/tizenrt/apps/examples/hello/hello_main.c:64
```

Comment:
Instead of printing "Address" and "Symbol" in each line, we must give a column heading to the table. 
In fact, we need to have 4 columns as follows:
`Dump address          Symbol address         Symbol name             File name`